com.ibm.streamsx.topology/opt/python/packages/streamsx/rest_primitives.py

```diff
@@ -618,11 +908,11 @@ def cancel_job(self, job_id=None, job_name=None):
         """Cancel a running job.
 
         Args:
-            job_id(str): Identifier of job to be canceled.
-            job_name(str): Name of job to be canceled.
+            job_id (str, optional): Identifier of job to be canceled.
```

Comment:
I got this from the google style doc: http://www.sphinx-doc.org/en/stable/ext/example_google.html  They use `optional` to indicate that an argument is optional.  (Though I agree the method signature gives the same information as well)